e2e/test/scripts/get-nb-config.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
```

Comment:
Might want to also fail on unset vars, I generally use:

```suggestion
set -euo pipefail
```